packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -132,8 +133,7 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
 
             return;
         } else if (selection?.type == 'image' && selection.image !== rawEvent.target) {
-            this.selectBeforeImage(editor, selection.image);
-
+            this.selectBeforeOrAfterElement(editor, selection.image);
```

Comment:
Is this change from Table features? If yes, could we make this change part of edit plugin?

Reply:
It could be done but I feel it is easier to implement in SelectionPlugin, since Arrow key movement is also implemented there. It is a good behaviour to have as a core feature.